static/xref/script.js

```diff
@@ -226,7 +226,7 @@ async function ready() {
   const allTypes = [].concat(...Object.values(types)).sort();
   updateInput(form.types, allTypes);
 
-  const fuse = new Fuse(terms, { caseSensitive: true });
+  const fuse = new Fuse(terms, {});
```

Comment:
Need to pass `{}`, otherwise Fuse cries.
Might upgrade Fuse to newer (major) version next